src/main/java/io/dashboardhub/pipelinedashboard/domain/User.java

```diff
@@ -19,14 +20,15 @@
     private String uid;
 
     @NotNull
-    @Size(min = 1, max = 255)
     private String username;
 
+    @Size(min = 5, max = 255)
     private String name;
 
-//    @ManyToMany
-//    @JoinTable(name="users_projects")
-//    private Set<Project> projects;
+    @Email
+    private String email;
+
+    private String lastLoggedIn;
```

Comment:
haha that is a unfortunate side effect of github user tagging - i bet their are lots of users that get a similar deal with all the java annotation. @eddiejaoude @-temporal is a java persistence annotation, not specific to spring, but to JPA